lib/src/components/NxSubmitMask/NxSubmitMask.scss

```diff
@@ -30,17 +31,18 @@
 
 .nx-submit-mask__message {
   @include container-vertical;
+  @include semi-bold();
 
-  background-color: $nx-off-white;
-  border: 2px solid $nx-slate;
-  box-shadow: -4px 4px 0px rgba($nx-slate, 0.65);
+  background-color: #fff;
+  border-radius: $nx-border-radius;
   margin: 0;
-  padding: 22px 12px;
+  padding: $nx-spacing-l;
+  min-width: 124px;
```

Comment:
This was intended as a border-box width but it's currently the content-box width

Reply:
> This was intended as a border-box width but it's currently the content-box width

Fixed here by changing it to `border-box`: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/12f0e8dbc13f4e55843306be3219208c4a9eaba3